Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -141,6 +143,22 @@ private void WriteSocialIntegrationState()
 		}
 		
 		public void OnSoomlaGUI() {
+			if (EditorUserBuildSettings.activeBuildTarget == 
+#if UNITY_5
+			    BuildTarget.iOS
+#else
+			    BuildTarget.iPhone
+#endif
```

Comment:
@vedi, I don't have Unity v4.5, only v4.6.
But `UNITY_5` is defined only in Unity v5, so for other versions flow will goes at the `else` condition (behaviour will be the same all non-v5 Unitys).